docs/reference/cli.md

```diff
@@ -41,40 +43,24 @@ will provide a link to download a preconfigured configuration file for you.
 `lakectl` configuration items can each be controlled by an environment variable. The variable name will have a prefix of
 *LAKECTL_*, followed by the name of the configuration, replacing every '.' with a '_'. Example: `LAKECTL_SERVER_ENDPOINT_URL` 
 controls `server.endpoint_url`.
-### lakectl
-
-A cli tool to explore manage and work with lakeFS
-
-#### Synopsis
-{:.no_toc}
 
-lakectl is a CLI tool allowing exploration and manipulation of a lakeFS environment
+## Running lakectl from Docker
 
-#### Options
-{:.no_toc}
+If you'd rather run `lakectl` from a Docker container you can do so by passing configuration elements as environment variables. Here is an example: 
 
-```
-      --base-uri string      base URI used for lakeFS address parse
-  -c, --config string        config file (default is $HOME/.lakectl.yaml)
-  -h, --help                 help for lakectl
-      --log-format string    set logging output format
-      --log-level string     set logging level (default "none")
-      --log-output strings   set logging output(s)
-      --no-color             don't use fancy output colors (default when not attached to an interactive terminal)
-      --verbose              run in verbose mode
+```bash
+$ docker run --rm --interactive --tty \
+          -e LAKECTL_CREDENTIALS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE \
+          -e LAKECTL_CREDENTIALS_SECRET_ACCESS_KEY=xxxxx
+          -e LAKECTL_SERVER_ENDPOINT_URL=https://host.us-east-2.lakefscloud.io/ \
+          --entrypoint lakectl treeverse/lakefs \
+          repo list
 ```
 
-**note:** The `base-uri` option can be controlled with the `LAKECTL_BASE_URI` environment variable.
-{: .note .note-warning }
+_Bear in mind that if you are running lakeFS itself locally you will need to account for this in your networking configuration of the Docker container. That is to say, `localhost` to a Docker container is itself, not the host machine on which it is running._
```

Comment:
This is an explanation of how networking in Docker works - not really relevant here IMO. If you want to mention it, use an example of how to run locally.

Reply:
I'm going to leave this in for pragmatic reasons (Docker/local always trips people up), and I've [logged an issue](https://github.com/treeverse/lakeFS/issues/5358) to add example of running it locally.